src/openrct2-ui/windows/Staff.cpp

```diff
@@ -290,6 +292,8 @@ static constexpr const uint32_t window_staff_page_enabled_widgets[] = {
 
 static uint8_t _availableCostumes[ENTERTAINER_COSTUME_COUNT];
 
+static int32_t s_staff_patrol_area_paint_value = -1;
```

Comment:
Rename to `_staffPatrolAreaPaintValue`.
Please have a look at https://github.com/OpenRCT2/OpenRCT2/wiki/Coding-Style